src/wp-includes/style-engine/class-wp-style-engine-css-declarations.php

```diff
@@ -0,0 +1,191 @@
+<?php
+/**
+ * WP_Style_Engine_CSS_Declarations
+ *
+ * Holds, sanitizes and prints CSS rules declarations
+ *
+ * @package WordPress
+ * @subpackage StyleEngine
+ * @since 6.1.0
+ */
+
+/**
+ * Class WP_Style_Engine_CSS_Declarations.
+ *
+ * Holds, sanitizes, processes and prints CSS declarations for the style engine.
+ *
+ * @access private
+ * @since 6.1.0
+ */
+class WP_Style_Engine_CSS_Declarations {
+
+	/**
+	 * An array of CSS declarations (property => value pairs).
+	 *
+	 * @since 6.1.0
+	 *
+	 * @var array
+	 */
+	protected $declarations = array();
+
+	/**
+	 * Constructor for this object.
+	 *
+	 * If a `$declarations` array is passed, it will be used to populate
+	 * the initial $declarations prop of the object by calling add_declarations().
+	 *
+	 * @since 6.1.0
+	 *
+	 * @param array $declarations An array of declarations (property => value pairs).
+	 */
+	public function __construct( $declarations = array() ) {
+		$this->add_declarations( $declarations );
+	}
+
+	/**
+	 * Add a single declaration.
```

Comment:
```suggestion
	 * Adds a single declaration.
```

Active not imperative (applies to all instances in the PR) :slightly_smiling_face: